Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.h

```diff
@@ -406,9 +406,9 @@ class AXIsolatedTree : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<AX
     void addUnconnectedNode(Ref<AccessibilityObject>);
     bool isUnconnectedNode(std::optional<AXID> axID) const { return axID && m_unconnectedNodes.contains(*axID); }
     // Removes the corresponding isolated object and all descendants from the m_nodeMap and queues their removal from the tree.
-    void removeNode(const AccessibilityObject&);
+    void removeNode(AXID, std::optional<AXID>);
```

Comment:
It could be nice to have a comment explaining what the second parameter is, e.g.:

```cpp
void removeNode(AXID, std::optional<AXID> /* parentID */);
```